basterdz-api/src/main/java/com/dnd/domains/room/controller/RoomController.java

```diff
@@ -0,0 +1,28 @@
+package com.dnd.domains.room.controller;
+
+import com.dnd.common.dto.ApiResult;
+import com.dnd.domains.room.dto.request.RoomCreateRequestDto;
+import com.dnd.domains.room.dto.response.RoomCreateResponseDto;
+import com.dnd.domains.room.service.RoomService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1")
+@RequiredArgsConstructor
+public class RoomController {
+
+    private final RoomService roomService;
+
+    @PostMapping("/rooms")
```

Comment:
```suggestion
@RestController
@RequestMapping("/api/v1/rooms")
@RequiredArgsConstructor
public class RoomController {

    private final RoomService roomService;

    @PostMapping
```
RoomController 엔드포인트가 어짜피 /rooms라면 이렇게 해도 될것 같습니다.

Reply:
수정했습니다!